agent/src/beerocks/slave/son_slave_thread.h

```diff
@@ -86,18 +86,14 @@ class slave_thread : public beerocks::socket_thread {
         STATE_WAIT_RETRY_CONNECT_TO_BACKHAUL_MANAGER,
         STATE_WAIT_FOR_BACKHAUL_MANAGER_REGISTER_RESPONSE,
         STATE_JOIN_INIT,
-        STATE_START_AP_MANAGER,
         STATE_WAIT_FOR_AP_MANAGER_JOINED,
-        STATE_AP_MANAGER_JOINED,
-        STATE_START_MONITOR,
         STATE_WAIT_FOR_MONITOR_JOINED,
-        STATE_MONITOR_JOINED,
-        STATE_WAIT_BEFORE_JOIN_MASTER,
         STATE_BACKHAUL_ENABLE,
         STATE_SEND_BACKHAUL_MANAGER_ENABLE,
         STATE_WAIT_FOR_BACKHAUL_MANAGER_CONNECTED_NOTIFICATION,
         STATE_BACKHAUL_MANAGER_CONNECTED,
         STATE_WAIT_BACKHAUL_MANAGER_BUSY,
+        STATE_WAIT_BEFORE_JOIN_MASTER,
```

Comment:
Shouldn't this be explained in a bit more details (and perhaps as a separate commit)?

Reply:
Not critical, I've just moved it to the right location on the enum. Nothing new has been added.